src/app.ts

```
@@ -77,6 +81,21 @@ app.use((req, res, next) => {
+// Initialize the database connection
+// AppDataSource.initialize()
+//   .then(() => {
+//     logger.info("Data Source has been initialized!");
+//   })
+//   .catch((err) => {
+//     logger.error("Error during Data Source initialization:", err);
+//     process.exit(1); // Exit if we can't connect to the database
+//   });
+
+app.use(((req: Request, res: Response, next: NextFunction) => {
+  req.dataSource = AppDataSource;
```

ESLint

[error] 'AppDataSource' is not defined.

(no-undef)